### PR TITLE
Create new search ranking blog

### PR DESCRIPTION
**Pull request by iskyOS**
Create new blog for the announcement of search ranking inspired by PageRank.

### CODE REVIEW

File: content/blogposts/2022/new-search-ranking.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+The #1 search result on the right is a match inside a source comment. Although this might be relevant in some contexts, it’s generally not so great to suggest as the top result.
+
+With the pageranked results on the left, you only see relevant matches. Over time, this adds up to tremendous time savings. And devs will always gravitate towards things that save them time.
+
+Why is using PageRank for Code Search so revolutionary and effective? Let’s dig in.
+
+<Figure src="https://storage.googleapis.com/sourcegraph-assets/blog/new-search-ranking/pagerank-visual.png" alt="Image depicting the PageRank algorithm" caption="Source: Wikipedia: User:Mayhatmate / CC-BY-SA-3.0" />
+
+In the illustration above, the size of each smiley is proportional to the number of other smileys pointing at it. For web pages, Google’s PageRank tracks which pages are pointed at (referenced) most often by other web pages. PageRank a measure of how “cool” they are: Who’s pointing at them?
```
Comment (abschu) [inline content/blogposts/2022/new-search-ranking.md:41]:
```suggestion
In the illustration above, the size of each smiley is proportional to the number of other smileys pointing at it. For web pages, Google’s PageRank tracks which pages are pointed at (referenced) most often by other web pages. PageRank is a measure of how “cool” they are: Who’s pointing at them?
```

File: content/blogposts/2022/new-search-ranking.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+### What’s such a big deal about all this?
+
+Just because you can implement PageRank doesn’t magically make you Google. Lookin’ at you, Bing.
+
+Google’s search quality comes from their knowledge graph. They have deep domain intelligence and thousands of hand-coded heuristics, to ensure you get what you’re searching for on the first try, every time.
+
+What’s the equivalent knowledge graph in code search? Why, that would be the Source Graph, of course. Yep, that’s right. In order to compute ranking on code, you run it on the source graph.
+
+That source graph is not a simple matter to create. We did it with our code intelligence data, which scales up to the world’s largest code bases.
```
Comment (abschu) [inline content/blogposts/2022/new-search-ranking.md:71]:
```suggestion
That source graph is not a simple matter to create. We did it with our code intelligence data, which scales up to the world’s largest codebases.
```

**Review by abschu — COMMENTED** [review]
Two minor edits + the CIP capitalization.

**Review by abschu — APPROVED** [review]
LGTM!

File: content/blogposts/2022/new-search-ranking.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+### Announcing Search Ranking and Relevance
+
+I’m thrilled to announce that Sourcegraph has launched PageRank-driven Code Search result rankings that prioritize relevance and showing reusable code. This launched today for searches on popular OSS repos on https://sourcegraph.com, and we are working to bring ranking to private Sourcegraph deployments soon.
+
+This launch ushers in a new era of search-result relevance, helping you find the code you’re looking for on the first try.
+
+Sourcegraph’s new search ranking uses a rendition of the Google [PageRank](https://en.wikipedia.org/wiki/PageRank) algorithm on source code, powered by the code symbol graph from our sophisticated code intelligence platform (CIP).
+
+Code Search is now relentlessly focused on helping you find **relevant** answers. Consider this side-by-side comparison from an early run, with pageranked results on the left.
```
Comment (olafurpg) [inline content/blogposts/2022/new-search-ranking.md:29]: What is the query? It's difficult to see from the screenshot

**Pull request merged**